astore/client/auth/auth.go

```diff
@@ -74,14 +74,18 @@ func (c *Client) Login(username, domain string, o LoginOptions) (string, error)
 	}
 	fmt.Printf("\t%s\n\nTo complete authentication with @%s.\n"+
 		"I'll be waiting for you, but hurry! The request may timeout.\nHit Ctl+C with no regrets to abort.\n", ares.Url, domain)
-	browser.OpenURL(ares.Url)
-	pubKey, privateKey, err := kcerts.MakeKeys(kcerts.GenerateDefault)
+	go func() {
+		if err := browser.OpenURL(ares.Url); err != nil {
+			o.Logger.Infof("error opening url, you must open it yourself")
+		}
+	}()
+	privateKey, pubKey, err := kcerts.MakeKeys(kcerts.GenerateDefault)
 	if err != nil {
 		return "", fmt.Errorf("error generating ssh credentials: %w", err)
 	}
 	treq := &auth.TokenRequest{
 		Url:       ares.Url,
-		Publickey: pubKey,
+		Publickey: ssh.MarshalAuthorizedKey(pubKey),
```

Comment:
no, this should not work? `kcerts.MakeKeys` now returns a marshalled key directly?

Reply:
as a side note, this function as is should not be working. Did you forget to push another commit with fixes? How did you test this?

Reply:
never mind, just noticed that MakeKeys is back to returning non-marshalled keys.